sdk/all/src/jmh/java/io/opentelemetry/sdk/metrics/MetricsTestOperationBuilder.java

```diff
@@ -23,102 +23,90 @@
  */
 public enum MetricsTestOperationBuilder {
   LongCounterAdd(
-      new OperationBuilder() {
-        @Override
-        public Operation build(final Meter meter) {
-          return new Operation() {
-            LongCounter metric = meter.longCounterBuilder("long_counter").build();
-            LongCounter.BoundLongCounter boundMetric =
-                meter
-                    .longCounterBuilder("bound_long_counter")
-                    .build()
-                    .bind(Labels.of("KEY", "VALUE"));
-
-            @Override
-            public void perform(Labels labels) {
-              metric.add(5L, labels);
-            }
-
-            @Override
-            public void performBound() {
-              boundMetric.add(5L);
-            }
-          };
-        }
+      meter -> {
+        return new Operation() {
```

Comment:
While we're addressing IDE warnings, what about the IDE suggestion of removing `return` too? Up to you though

Reply:
Did not find that, but I will follow-up